cg/cli/set/base.py

```diff
@@ -207,22 +206,23 @@ def show_option_help(short_name: str = "", long_name: str = "", help_text: str =
     multiple=True,
     help=HELP_KEY_VALUE,
 )
+@click.option("-lk", "--listkeys", is_flag=True, help="List all available modifiable sample properties")
 @click.option(OPTION_LONG_SKIP_LIMS, is_flag=True, help=HELP_SKIP_LIMS)
 @click.option(OPTION_SHORT_YES, OPTION_LONG_YES, is_flag=True, help=HELP_YES)
-@click.option("--help", is_flag=True)
 @click.pass_obj
 def sample(
     context: CGConfig,
     sample_id: Optional[str],
```

Comment:
If you end up doing `cg set list`, then I would change sample_id as a required option instead. Something like:
```
@click.option("-s", "--sample-id", required=True, help="Internal sample ID")
```

Reply:
Nice idea, it has been implemented :)